agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/DiseaseAnnotationCurationIndexer.java

```diff
@@ -133,7 +134,11 @@ private List<GeneDiseaseAnnotationDocument> createGeneDiseaseAnnotationDocuments
 					gdad.setParentSlimIDs(closureMap.get(da.getObject().getCurie()));
 					lookup.put(key, gdad);
 				}
-				gdad.setEvidenceCodes(da.getEvidenceCodes());
+				
```

Comment:
So the current list that is in the `GeneDiseaseAnnotationDocument gdad` needs to be added to, not overridden. So you need to take the list that is in gdad and add those to the map first then add the ones from da and do the setEvidenceCodes.